module/Finna/src/Finna/AjaxHandler/FeedTrait.php

```diff
@@ -105,7 +105,7 @@ protected function formatFeed(
         if (isset($config->title)) {
```

Comment:
Koska tämä on vielä vähän epäselvä, niin ehkä tässä voisi olla nyt jotain tyyliin:
```
$title = $config->title ?? 'rss';
if ('rss' === $title) {
    $feed['title'] = $channel->getTitle();
    $feed['translateTitle'] = false;
} else {
    $feed['title'] = $title;
    $feed['translateTitle'] = true;
}

```

Näin olisi vain yksi title, ja sitten translateTitle-boolean kertoisi, pitääkö se kääntää. Kukin templaatti sitten määrittelee itse, missä ja miten titlen näyttää.